e2e/lvmd3.yaml

```diff
@@ -5,5 +5,10 @@ device-classes:
     default: true
     spare-gb: 1
   - name: "hdd2"
+    volume-group: "node3-myvg2"
+    spare-gb: 1
+  - name: "raid"
     volume-group: "node3-myvg3"
     spare-gb: 1
+    lvcreate-options:
+      - "--type=raid1"
```

Comment:
Shouldn't "hdd2" still use "node3-myvg3" and "raid" use "node3-myvg4"?

Reply:
I'm not sure why hdd2 was using node3-myvg3 and there was no node3-myvg2 so I changed it so that hdd2 is similar to the other (lvmd1.yaml and lvmd2.yaml). Is there a reason to have this different for lvmd3?
By changing it to node3-myvg2 it was possible to simplify some things in the Makefile. [This is how it was](https://github.com/topolvm/topolvm/blob/27ca8eeeb578d4c8aa7bedd93039cc630ee54e8d/e2e/Makefile#L126-L135) and then I changed it like [this](https://github.com/Nordix/topolvm/blob/0549a9a3dce5399b1d7353fd12144261e5fa2191/e2e/Makefile#L121-L126).

Reply:
@lentzi90 

> I'm not sure why hdd2 was using node3-myvg3 and there was no node3-myvg2 so I changed it so that hdd2 is similar to the other (lvmd1.yaml and lvmd2.yaml). Is there a reason to have this different for lvmd3?

I'm asking the reason to the original author. Please wait for a while.

Reply:
@lentzi90 He said "it's by mistake", haha. So please keep this code as-is and please reflect the @cupnes 's comment. We'll merge this PR after that.

Reply:
Thank you!